Fixed multiple flake8

Dimi20cen opened the pull request:
- I didn't fix `W504` flakes when the line was a comment because I was afraid that it might negatively affect readability.

- In line 428 of `lib/myelements/add_objects.py` there's a variable `radius` which is undefined, I decided to not handle it in this PR.

- In `lib/myelements/add_objects.py` , there's also a variable named 'l' that represents the length of a vector and it causes a `E741` (ambiguous variable name). I didn't know if it was in the scope of this PR to change it so I left it as is.

@chimosky, kindly review

Dimi20cen committed:
Fixed multiple flake8

chimosky commented:
> * I didn't fix `W504` flakes when the line was a comment because I was afraid that it might negatively affect readability.
> 
>     * In line 428 of `lib/myelements/add_objects.py` there's a variable `radius` which is undefined, I decided to not handle it in this PR.
> 
>     * In `lib/myelements/add_objects.py` , there's also a variable named 'l' that represents the length of a vector and it causes a `E741` (ambiguous variable name). I didn't know if it was in the scope of this PR to change it so I left it as is.
> 


Yes both are within the scope of flake8 as you noticed then when testing for flake8 errors.

Dimi20cen committed:
Fix flake8

- Configure comment lines and print statements to fix flake8, W504
- Rename variable l that was ambiguously named
- Replace `except` with `except Exception` to fix flake8, E722

Dimi20cen committed:
Fix flake8

- Remove line `circleShape.radius = radius`, line 436 in add_objects.py, since variable radius was not defined and caused flake8, F821

chimosky commented on lib/myelements/add_objects.py:
```diff
@@ -58,7 +58,8 @@ def triangle(
             friction=0.5,
             screenCoord=True):
-        """ Add a triangle | pos & a in the current input unit system (meters or pixels)
+        """ Add a triangle | pos & a in the current input unit system
+            (meters or pixels)
```
This would be better if the quotes are on its own line, same with every other one like it.

Dimi20cen replied:
Sure, fixed

chimosky commented on lib/myelements/add_objects.py:
```diff
@@ -530,17 +540,21 @@ def convexPoly(
 
-        print("convexPoly: Polygon reduced from %i to %i vertices | tolerance: %i" % (
-            len(vertices), len(v_new), tolerance))
+        msg = ("convexPoly: Polygon reduced from %i to %i vertices | "
+               "tolerance: %i")
+        print(msg % (len(vertices), len(v_new), tolerance))
+
```
I'm guessing this was a debug need, you can use git blame and look at the particular commit that introduced it and see if there's any explanation and if there isn't then you can use logging.debug for this if it's used in other places or just remove it entirely.

Dimi20cen committed:
Multiline comments configuration

In multiline comments, start the comment in a new line below the three quotes
